frontends/web/src/routes/settings/components/settingsItem/settingsItem.module.css

```diff
@@ -30,9 +30,8 @@
 
 
 .rightContentContainer {
-    display: flex;
     align-items: center;
-    flex-shrink: 0;
+    display: flex;
 }
```

Comment:
fixes long text breaking weirdly in settings item:

![Screen Shot 2024-03-14 at 19 14 13](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/86ae5843-f117-40ef-97a4-d86939f6967f)

with this change:

![Screen Shot 2024-03-14 at 19 14 03](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/8df9c905-8ec6-4ac1-9699-6a9785c83eb9)